README.md

```diff
@@ -87,97 +93,141 @@ However, this is always optional, and usually not necessary.
 --
----@param path? string path to explicitly specify the notebook
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zkindex
-require("zk").index(path, options)
+require("zk.commands").index(options, path)
 
 ---Creates and opens a new note
 --
----@param path? string path to explicitly specify the notebook
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zknew
-require("zk").new(path, options)
+require("zk.commands").new(options, path)
 
 ---Creates a new note and uses the last visual selection as the title while replacing the selection with a link to the new note
 --
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+require("zk.commands").new_from_title_selection(options, path)
+
+---Creates a new note and uses the last visual selection as the content while replacing the selection with a link to the new note
+--
 ---@param options table additional options
-require("zk").new_link(path, options)
+---@param path? string path to explicitly specify the notebook
+require("zk.commands").new_from_content_selection(options, path)
+
+
+---cd into the notebook root
+require("zk.commands").cd(path)
 
 ---Opens a Telescope picker
 --
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").notes(options, path)
+
+---Opens a Telescope picker
+--
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
-require("zk").list(path, options)
+require("zk.commands").backlinks(options, path)
 
 ---Opens a Telescope picker
 --
+---@param options table additional options
+---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").links(options, path)
+
+---Opens a Telescope picker, filters for notes that match the text in the last visual selection
+--
+---@param options table additional options
 ---@param path? string path to explicitly specify the notebook
+---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zklist
+require("zk.commands").match(options, path)
+
+---Opens a Telescope picker
+--
 ---@param options table additional options
+---@param path? string path to explicitly specify the notebook
 ---@see https://github.com/mickael-menu/zk/blob/main/docs/editors-integration.md#zktaglist
-require("zk").tag.list(path, options)
+require("zk.commands").tags(options, path)
```

Comment:
I think keeping `zk.tag.list()` would be better as I will add more `zk tag` commands later on (like `rename` or `remove`).

Reply:
For consistency. The is zk.list command became zk.notes, because that fits better when the other commands are called zk.backlinks etc. Having that, I think zk.tags fits better with zk.notes. For rename, remove we could add zk.tag_rename or zk.rename_tag.

Reply:
Okay, that makes sense. So I guess we can consider that the "edit/open" action is the default for every functions named as nouns?